### DISCUSSION TITLE
Is it able to click the workspace in the bar to back_and_forth?

### DISCUSSION THREAD

**Discussion by c02y** [i3/i3#5761] [Q&A] [answered]
When I'm in workspace 3, I can use `mod+3` to switch to the previous focused workspace such as workspace 1, and `mod+3` in workspace 1 to switch to workspace 3 back again. (If I set `workspace_auto_back_and_forth yes` in i3/config)

Is it able to click the workspace in the bar for this feature using my mouse?

**Comment by orestisfl** [comment, marked as answer]
It is a 4.23 issue: https://github.com/i3/i3/issues/5744

**Reply by c02y** [reply]
No wonder I miss this, it turns out it was removed accidentally somehow.
